src/com_tjvendors/admin/models/payout.php

```diff
@@ -236,9 +236,6 @@ public function save($data)
 			// Update their details in the users table using id as the primary key.
 			$result = Factory::getDbo()->updateObject('#__tjvendors_passbook', $payout_update, 'id');
 
-			$message = Text::_('COM_TJVENDORS_PAYOUT_SUCCESSFULL_MESSAGE');
```

Comment:
@deepa-g Do we set a success message in the controller for this? Is this tested in case we do bulk payout from TJ-Vendors?

Reply:
Success message handled in core Joomla form controller